docs/guided-remediation.md

```diff
@@ -243,11 +302,320 @@ osv-scanner fix --non-interactive --format json --strategy=relax -M path/to/pack
+          "versionFrom": "3.2.1",
+          "versionTo": "3.2.2",
+          "transitive": true
+        }
+      ],
+      "fixed": [
+        {
+          "id": "GHSA-6hgm-866r-3cjv",
+          "packages": [
+            {
+              "name": "commons-collections:commons-collections",
+              "version": "3.2.1"
+            }
+          ]
+        },
+        {
+          "id": "GHSA-fjq5-5j5f-mvxh",
+          "packages": [
+            {
+              "name": "commons-collections:commons-collections",
+              "version": "3.2.1"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "packageUpdates": [
+        {
+          "name": "commons-io:commons-io",
+          "versionFrom": "2.1",
+          "versionTo": "2.14.0",
+          "transitive": false
+        }
+      ],
+      "fixed": [
+        {
+          "id": "GHSA-78wr-2p64-hpwj",
+          "packages": [
+            {
+              "name": "commons-io:commons-io",
+              "version": "2.1"
+            }
+          ]
+        },
+        {
+          "id": "GHSA-gwrp-pvrq-jmwv",
+          "packages": [
+            {
+              "name": "commons-io:commons-io",
+              "version": "2.1"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "packageUpdates": [
+        {
+          "name": "commons-beanutils:commons-beanutils",
+          "versionFrom": "1.7.0",
+          "versionTo": "1.9.4",
+          "transitive": true
+        }
+      ],
+      "fixed": [
+        {
+          "id": "GHSA-6phf-73q6-gh87",
+          "packages": [
+            {
+              "name": "commons-beanutils:commons-beanutils",
+              "version": "1.7.0"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "packageUpdates": [
+        {
+          "name": "commons-net:commons-net",
+          "versionFrom": "1.4.1",
+          "versionTo": "3.9.0",
+          "transitive": false
+        }
+      ],
+      "fixed": [
+        {
+          "id": "GHSA-cgp8-4m63-fhh5",
+          "packages": [
+            {
+              "name": "commons-net:commons-net",
+              "version": "1.4.1"
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+</details>
+
 ## Scripted usage
 
 The `--non-interactive` mode, in combination with [other flags](#remediation-flags), can be used in scripts to automatically apply and test remediation patches.
 
-Check out our [sample Python script](https://github.com/google/osv-scanner/blob/main/scripts/examples/auto_guided_remediation.py) that uses `osv-scanner fix` to remediate as many vulnerabilities as possible without failing your project's `npm run test`.
+Check out our [sample Python script](https://github.com/google/osv-scanner/blob/main/scripts/examples/auto_guided_remediation.py) that uses `osv-scanner fix` to remediate as many vulnerabilities as possible in an npm project without failing your project's `npm run test`.
```

Comment:
under "Interactive mode", should we specify that it only works for npm at the moment?